pkg/processor/batchprocessor_test.go

```diff
@@ -112,157 +110,6 @@ func TestPollSendSuccess(t *testing.T) {
 	assert.True(t, strings.Contains(log.Logs[1], doneMsg))
 }
 
-func TestPollSendFailedOnceMoreThanMin(t *testing.T) {
```

Comment:
We removed quite a few tests because of the new functionality.

Can we add some back in that test the new functionality?

Reply:
With the removal of custom backoff logic, we automatically fall to the AWS SDK retry behavior which is not a new functionality in our code base. Are you suggesting we add a unit test for the AWS SDK retry functionality? IMO it won't be appropriate because if their retry behavior changes, it will break our tests.